### ISSUE TITLE
Support for GNU style NEWS and Changelog files

### ISSUE THREAD

**Issue by mihaigiurgeanu** [haskell/hackage-server#691]
In the GNU style, the history of user visible changes it is kept in a file named NEWS and the Changelog file contains a history of finner, code-level changes made to the code.

My proposal is to support this style by detecting if NEWS and Changelog files are included in the package and, if they are, to display the NEWS file content instead of Changelog.

**Comment by gbaz** [comment]
Why not display both?

**Comment by hvr** [comment]
I agree w/ Gershon; we should definitely always provide a link to the ChangeLog file (if it exists), and *additionally* if a `NEWS` is found display a link to that one as well (as plain-text though; afaik NEWS file have no well-defined markup) -- unless we add a specific handler for NEWS files, it'd be enough to simply generate a non-rendered link into the source-folder of that tarball.

**Comment by mihaigiurgeanu** [comment]
Yes, I agree, it makes no sense to display the NEWS instead of Changelog, they should both be displayed if they exists
